salt/modules/zabbix.py

```diff
@@ -1058,6 +1058,97 @@ def host_update(hostid, **connection_args):
         return ret
 
 
+def hostinventory_get(hostids, **connection_args):
+    '''
+    Retrieve host inventory according to the given parameters.
+    See: https://www.zabbix.com/documentation/2.4/manual/api/reference/host/object#host_inventory
+
+    .. versionadded:: Oxygen
```

Comment:
This should be updated to `Fluorine` as we've completed the feature freeze for `oxygen` already.